src/main.ts

```diff
@@ -109,13 +112,13 @@ export type WrappedScheduledFunction = (
 export function wrap<T>(
   cloudFunction: CloudFunction<T>
 ): WrappedScheduledFunction | WrappedFunction {
-  if (!has(cloudFunction, '__trigger')) {
+  if (!cloudFunction.__trigger) {
     throw new Error(
       'Wrap can only be called on functions written with the firebase-functions SDK.'
     );
   }
 
-  if (get(cloudFunction, '__trigger.labels.deployment-scheduled') === 'true') {
+  if (cloudFunction?.__trigger?.labels && cloudFunction?.__trigger?.labels['deployment-scheduled'] === 'true') {
```

Comment:
`if (cloudFunction.trigger?.labels?.["deployment-scheduled"] === 'true') {`

Reply:
Done (thanks for calling out the improved syntax!)